testData/execReturn/foo.jet

```diff
@@ -0,0 +1,4 @@
+{{ f := "f" }}
+{{ o := "o" }}
+... some content that will be discarded when this template runs inside exec() ...
+{{ return f+o+o }}
```

Comment:
Nit: can you add a trailing newline?

Reply:
0085ed7 :+1:

Reply:
And now your tests are failing because of the added newlines. I guess we don't want those, right?

Reply:
We don't want the newlines or we don't want the test failures? :D
Anyway, I somehow had to change only one of the tests... is this expected behavior?
db4d56e